.github/workflows/ci.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       matrix:
         build_type: [Release]
         os: [ubuntu-latest, macOS-latest, windows-2019]
-        project_tags: [Default, Unstable, Release202005feat01, Release202008patch01]
+        project_tags: [Default, Unstable, Release202011]
```

Comment:
Did you reduce the releases tested from 2 to 1 for build time issues?

Reply:
No, we remove the old releases as they will break when we add the portaudio dependencies, see https://github.com/robotology/robotology-superbuild/pull/499#issuecomment-712121774 for more details.